### ISSUE TITLE
Errors when building in mbed studio

### ISSUE THREAD

**Issue by Hphipps** [ARMmbed/mbed-os-example-aws#3]
Errors when building in mbed studio.

**Comment by 0xc0170** [comment]
@Hphipps please provide more details. what errors are there? What version of tools you use? Anything that will help to reproduce it locally.

**Comment by LDong-Arm** [comment]
With Arm Compiler 6.13 (same one in Mbed Studio) and CLI:
```
Compile [  4.8%]: NanostackRfPhyAT86RF215.cpp                                                                                                                                                                     
[Error] version@1,1: expected unqualified-id
```
There's no issue with GCC_ARM.

I think this is what happened? @0xc0170

**Comment by LDong-Arm** [comment]
Hi @Hphipps, we've fixed the issue. Would you please try a clean fetch and build?

(This issue was closed automatically by GitHub as we merged #7)

**Comment by Hphipps** [comment]
Builds with no issues now. Thanks for the fix.
